apm-sniffer/bootstrap-plugins/jdk-http-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdk/http/HttpClientNewInstanceInterceptor.java

```diff
@@ -18,23 +18,23 @@
 
 package org.apache.skywalking.apm.plugin.jdk.http;
 
+import java.lang.reflect.Method;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
-import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.StaticMethodsAroundInterceptor;
-
-import java.lang.reflect.Method;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.v2.MethodInvocationContext;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.v2.StaticMethodsAroundInterceptorV2;
 
-public class HttpClientNewInstanceInterceptor implements StaticMethodsAroundInterceptor {
+public class HttpClientNewInstanceInterceptor implements StaticMethodsAroundInterceptorV2 {
 
     @Override
     public void beforeMethod(Class clazz, Method method, Object[] allArguments, Class<?>[] parameterTypes,
-        MethodInterceptResult result) {
+                             MethodInterceptResult result, MethodInvocationContext context) {
```

Comment:
I am feeling `MethodInterceptResult` and `MethodInvocationContext` should be merged, as result to override the execution result is a case of MIC usages.